recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
I am really concerned about stability here too. I think switching back to an older version just because not to use `svn` is not a good idea here. ArchLinux already packages `10.69.02` and Fedora packages (depending on distribution version) are between `10.66` and `10.71`. Ubuntu and Debian use their own fork of `netpbm` called `netpbm-free`. The latest tarball release is `10.47`. Don't you think these distributions had a compelling reason to package an `unstable` version as you say?